pkg/reconciler/kubernetes/tektonresult/tektonresult.go

```diff
@@ -267,9 +267,22 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tr *v1alpha1.TektonResul
 		return v1alpha1.REQUEUE_EVENT_AFTER
 	}
 
-	// Mark InstallerSet Ready
+	// MarkInstallerSetReady
 	tr.Status.MarkInstallerSetReady()
 
+	if err := r.extension.PostReconcile(ctx, tr); err != nil {
+		msg := fmt.Sprintf("PostReconciliation failed: %s", err.Error())
```

Comment:
on ` MarkPostReconcilerFailed` already formatting with `"PostReconciliation failed with message: %s", msg)`. 
can you remove this formatting?

Reply:
Sure.

Reply:
Done. Remove that from MarkPostReconcilerFailed.